### ISSUE TITLE
Add request queues

### ISSUE THREAD

**Issue by vania-pooh** [aerokube/moon#88]
Same stuff like in Selenoid.

**Comment by baudren** [comment]
Hello,

do I understand this correctly: there is no way, at the moment, to queue extraneous requests for moon?

So if I launch 5 tests in parallel on a grid, with only 4 available nodes, the 5th will be dropped and never played?

**Comment by aandryashin** [comment]
No, it will be queued and wait for available resources. This ticket is for
changing internal implementation of queues...

чт, 4 апр. 2019 г., 10:24 Benjamin Audren <notifications@github.com>:

**Comment by baudren** [comment]
Ok - I must be doing something bad with my deployment then. I use the free 4 pods license for moon on my openshift cluster, and if I go over the 4 pods limit, my jobs are dropped instead of queued. Is there a setting somewhere to turn the queuing on? I could find nothing in the documentation.

**Comment by aandryashin** [comment]
Moon does not drop requests. There is route in OpenShift, you have to tune
its timeout and set for ex 30 min...

чт, 4 апр. 2019 г., 10:35 Benjamin Audren <notifications@github.com>:

**Comment by baudren** [comment]
You are right, sorry about this! Setting a high timeout solves my issue. Thanks for the support!
